pom.xml

```diff
@@ -55,6 +55,7 @@
                          locale. Turkish is a notoriously tricky locale. -->
         <surefire.argline>@{argLine} -Duser.language=TR -Duser.country=tr</surefire.argline>
         <kotlin.version>1.9.23</kotlin.version>
+        <json.version>20240303</json.version>
```

Comment:
Besides the pretty printing, what do you need this library for?

Reply:
Directly for the pretty print. Indirectly to ensure that the JSON output is valid when using the `JSONObject` and `JSONArray` classes.